package.json

```diff
@@ -12,7 +12,7 @@
     "react-player": "^2.6.2",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.4",
-    "travis": "1.8.10"
+    "travis": "^0.1.1"
```

Comment:
@carlymclift since this file changed I would expect the package-lock.json file to have changed as well.  Maybe this helps? https://stackoverflow.com/questions/57867267/how-to-update-package-lock-json-without-doing-npm-install

Reply:
Passing Travis fine, I was mistaken.